content/save-load-arrays.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>I think there's a formatting issue in the last line (but could be ReviewNB that is creating a problem)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>No, that's a markdown issue one too many backticks, I'll correct to:</p><p> </p><p>&gt;<code>delimiter = ','</code>: this option tells <code>savetxt</code> to place a comma between each column in the file</p>